### DISCUSSION TITLE
Wrong episodes rename: Grey's anatomy to Station 19

### DISCUSSION THREAD

**Discussion by effemmeffe** [Flexget/Flexget#3189] [Q&A]
My flexget keeps renaming greys anatomy episodes as they were station 19 show.
How do I fix this?

The log: https://pastebin.com/wHa0j2Ch
My config: https://github.com/effemmeffe/flexget

**Comment by lamski** [comment]
Maybe you don't need to use `trackt` in the `move-*` tasks? Once flexget downloads a show, the series metainfo is stored in the database.

**Reply by effemmeffe** [reply]
But then I have to use the standardized metainfo if I want to create my directory structure as show_name (show_year)/show_season/episode_name, am I right?
What will be the advantage of doing that?

**Reply by lamski** [reply]
The advantage would be eliminating extra calls to trackt for meta info that you already have.

**Reply by effemmeffe** [reply]
Understood, I'll give a try, thanks.
